### PR TITLE
Updating youtube-guidelines.md with description notes

### LINKED ISSUE #4589: Update Youtube guidelines
The following location needs to be updated with the below bullet items:

Location: https://github.com/kubernetes/community/blob/abc269477475538409a6f4e6e390684c6fbf29e2/communication/youtube/youtube-guidelines.md

Update Items:
-[] Note that date must be in title and description
-[] Note that the date format must be YYYYMMDD
-[] Note that the section titled `Is this video made for kids?` should not be altered for search reasons

### PR DESCRIPTION
**Pull request by markyjackson-taulia**
Signed-off-by: markyjackson-taulia <marky.r.jackson@gmail.com>

**Which issue(s) this PR fixes**:

Fixes #4589

### CODE REVIEW

File: communication/youtube/youtube-guidelines.md [hunk 1]
```diff
@@ -174,6 +174,11 @@ Example description:
 Kubernetes Contributor Experience Special Interest Group Weekly Meeting. Check
 here (link to sig list) for a complete list of SIGs and when they meet. Join us!
 
+Please note the following items:
+* The date must be in title and description
+* The date format must be YYYYMMDD
+* The section titled `Is this video made for kids?` should not be altered for search reasons
+
```
Comment (jeefy) [inline communication/youtube/youtube-guidelines.md]: Could you add a couple examples of Good vs Bad titles?

Comment (markyjackson-taulia) [reply communication/youtube/youtube-guidelines.md]: Yup, let me do that now

**Comment by mrbobbytables** [comment]
/approve
Thanks @markyjackson-taulia

**Pull request merged**